google_api_availability_platform_interface/lib/src/google_api_availability_platform_interface.dart

```diff
@@ -33,6 +34,13 @@ abstract class GoogleApiAvailabilityPlatform extends PlatformInterface {
     _instance = instance;
   }
 
+  /// Removes the registered instance of the [GoogleApiAvailabilityPlatform] class.
+  ///
+  /// This method is exposed for testing purposes only and should not be used by
+  /// clients of the plugin.
+  @visibleForTesting
+  static removeInstance() => _instance = null;
```

Comment:
Is there a specific use case for this? 

I have not seen it in any of the other plugins and from the top of my head I cannot really think of a specific case where this is needed. Would like to hear if there is one.

Reply:
It is included so we can test whether the methods that are exposed to users of the plugin correctly throw if no platform is registered. This way we can emulate the scenario where the app runs on an unsupported platform.